src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -281,6 +281,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                      SyntaxKind.ExitWhileStatement,
                      SyntaxKind.ExitFunctionStatement,
                      SyntaxKind.ExitSubStatement,
+                     SyntaxKind.ExitOperatorStatement,
```

Comment:
There is no test for this case. It's not semantically valid, but has syntax, and working out the details of testing it will cost an hour I don't have today. Needs testing in the future as part of comprehensive testing of all syntactic constructs.